### ISSUE TITLE
IPv6 only Switches can not be discovered with default config

### ISSUE THREAD

**Issue by kampa133** [netdisco/netdisco#699]
Error:
info discover: status defer: deferred job with no device creds

Workaround; adding IPv6 address or networks to the device_auth section.

```
device_auth:
  - tag: default_v2_readonly
    community: public
    read: true
    write: false
  - tag: ipv6_test
    community: public
    read: true
    write: false
    only:
      - 2001:db8::/32
```

**Comment by rc9000** [comment]
@ollyg and other developers: I had a look at this and AFAICT it boils down to 'any' (which is injected into check_acl for stanzas without only:/no:) being a valid RFC1884 IPv4 notation in IP::Netaddr::Lite [around these bits](https://github.com/netdisco/netdisco/blob/c5140e206174a402ce144fbbf606fdcf446f824b/lib/App/Netdisco/Util/Permission.pm#L211), but this doesn't cover IPv6.

I have updated the [device_auth](https://github.com/netdisco/netdisco/wiki/Configuration#device_auth) doc for now, but in general we should make `any` also mean `::/0`. I might be able to implement and test this at some point, but other (quicker) PRs are of course welcome.

**Comment by ollyg** [comment]
Hi, @kampa133 thanks for the bug report. I have applied a patch and released 2.44.15 to CPAN

[master ab6305b4] default access list for device_auth should be ipv4 and ipv6
